### PR TITLE
Add reports app link

### PR DESCRIPTION
**Pull request by smnkrt**
**Changes:** 
 - add reports_url field to sec_config.yml.sample
 - add link in layout header

### CODE REVIEW

File: app/views/layouts/_header.haml [hunk 1]
```diff
@@ -23,6 +23,7 @@
               %ul.dropdown-menu
                 %li{ class: menu_class('roles') }= link_to "Roles", roles_path
                 %li{ class: menu_class('abilities') }= link_to "Abilities", abilities_path
+                %li{ class: menu_class('abilities') }= link_to "Reports", AppConfig.reports_url, target: '_blank'
```
Comment (netguru-hound) [inline app/views/layouts/_header.haml]: Line is too long. [113/80]

Comment (mamut) [inline app/views/layouts/_header.haml]: don't you need a different 'menu_class' for it?

Comment (smnkrt) [inline app/views/layouts/_header.haml]: I don't need a menu class there at all. Thanks for catching that.

**Comment by mamut** [comment]
lgtm!

**Pull request merged**